src/EditorFeatures/Core.Wpf/QuickInfo/OnTheFlyDocsView.xaml.cs

```diff
@@ -137,6 +137,19 @@ private async Task SetResultTextAsync(ICopilotCodeAnalysisService copilotService
 
         try
         {
+            if (_onTheFlyDocsElement.IsContentExcluded)
+            {
+                SetResultText(EditorFeaturesResources.Your_organization_excluded_a_referenced_file_from_Copilot);
+                CurrentState = OnTheFlyDocsState.Finished;
+                Logger.Log(FunctionId.Copilot_On_The_Fly_Docs_Content_Excluded, KeyValueLogMessage.Create(m =>
+                {
+                    m["SymbolHeaderText"] = _onTheFlyDocsElement.SymbolSignature;
```

Comment:
Not sure we should log details like this, knowing we couldn't display the link due to content exclusion is sufficient